src/firmware/olaf_control/olaf_control.ino

```diff
@@ -0,0 +1,175 @@
+/* Firmware for IARRC 2017
+   Author: Vincent Yuan, 
+   Modified: Nick Wu
+   Modified: James Asefa
+   Modified: Jinhao Lu
+   Modified: Gareth Ellis
+   Modified: Valerian Ratu
+   Date Last Modified: October 22, 2017
+*/
+
+/*
+ * UBC Snowbots - IARRC 2017
+ * Firmware for Control of an RC car
+ * 
+ * This firmware will take in a message of the form:
+ * `B<linear x><linear y><linear z><angular x><angular y><angular z>`
+ * where each `<>` is a single byte. The degree to rotate the servo controlling
+ * the front wheels is determined from `angular z`, and the speed of the car by `linear x`
+ * (the rest of the values are discarded)
+ */
+
+#include <SoftwareSerial.h>
+#include <stdlib.h>
+#include <Servo.h>
+
+// Uncomment this to enable debug message logging - messages sent over serial will be echoed back
```

Comment:
Should warn people that if DEBUGGING is activated, the robot might stop working due to timing errors.

Reply:
done